python/private/stage1_bootstrap_template.sh

```diff
@@ -186,7 +235,7 @@ command=(
 #
 # However, when running a zip file, we need to clean up the workspace after the
 # process finishes so control must return here.
-if [[ "$IS_ZIPFILE" == "1" ]]; then
+if [[ "$use_exec" == "0" ]]; then
```

Comment:
I think the comment should be updated to mention the above logic.

Reply:
Done